puppet.py

```diff
@@ -27,6 +27,23 @@ def check_disabled():
         sudo('test -f {0} && echo DISABLED || echo ENABLED'.format(lockfile))
 
 @task
+def regenerate_certificate():
+    """Regenerate a node certificate"""
+    if len(env.hosts) > 1:
+      abort("This task should only be run on one host at a time.")
+    env.hosts[0]
```

Comment:
Actually, I think this is either erroneously here or was meant to be the value of a variable. Either way, it can probably go. Good spot!